packages/sdk/src/restclient.ts

```diff
@@ -119,6 +119,10 @@ interface WasmError {
 export interface TxsResponse {
   readonly height: string;
   readonly txhash: string;
+  /** 🤷‍♂️ */
```

Comment:
This is vaguely equal to the module that defined the error.
So there is one for all the generic errors (invalid address), but then if wasmd defines a specific error type, and slashing another, these should have those codespaces on them.

`(Codespace, code)` needs to be a universally unique identifier